yarn-project/aztec.js/src/wallet/wallet.ts

```diff
@@ -187,7 +187,7 @@ export type Wallet = {
   sendTx(exec: ExecutionPayload, opts: SendOptions): Promise<TxHash>;
   createAuthWit(
     from: AztecAddress,
-    messageHashOrIntent: Fr | Buffer<ArrayBuffer> | IntentInnerHash | CallIntent,
```

Comment:
This change was needed after the Zod change. But it made the test fail so I have to look into it.